css/zoom.css

```diff
@@ -35,3 +35,11 @@ img.zoom-img {
   filter: "alpha(opacity=100)";
   opacity: 1;
 }
+.zoom-overlay-open {
```

Comment:
actually not sure you want to add this to the whole body – think it's better to just have the cursor change on the image